### DISCUSSION TITLE
Just curious: What's stopping it?

### DISCUSSION THREAD

**Discussion by abu-matterize** [XiaoMi/ha_xiaomi_home#225] [Q&A]
Hello there,

I was just wondering, what's stopping this integration to be the part of official ha integration: https://github.com/home-assistant/core/tree/dev/homeassistant/components

Or is it already planned as PR?

**Comment by SusanPhevos** [comment]
xiaomi_home has become an open-source project just three days. We have no plan to make it to be an official HA integration.😃

**Reply by abu-matterize** [reply]
Okay. It's the choice, no hurdles?

**Reply by SusanPhevos** [reply]
Honestly speaking, we didn't think of doing that.

**Reply by abu-matterize** [reply]
For now, or no plans at all?

**Comment by FuJuntao** [comment]
No rush in my opinion. I'm pretty satisfied with hacs installation
